firecracker/packer/build-rootfs.pkr.hcl

```diff
@@ -0,0 +1,119 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.6" # This is when `ami_org_arns` was added
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "instance_type" {
+  description = "The type of instance used to *build* the image"
+  type        = string
+  # This is the value that Buildkite uses in their Packer template
+  default = "m5.xlarge"
+}
+
+variable "region" {
+  description = "The AWS region in which the base AMI is found *and* where the new AMI will be created"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "aws_profile" {
+  description = "The AWS connection profile to use when creating the image"
+  type        = string
+  default     = env("AWS_PROFILE")
+}
+
+variable "dist_folder" {
+  description = "Location of grapl_root/dist"
```

Comment:
Might want to mention that this is the directory on the local workstation that the final image file will be deposited into.